### DISCUSSION TITLE
what is the motivation behind renaming `purge` to `content`?

### DISCUSSION THREAD

**Discussion by ikhattab** [tailwindlabs/tailwindcss#6397] [General]
what is the motivation behind renaming `purge` to `content` in `tailwind.config.js
`?

**Comment by surjithctly** [comment]
From the upgrade guide:

> _Since Tailwind no longer uses PurgeCSS under the hood, we’ve renamed the purge option to content to better reflect what it’s for:_

In short, its using JIT mode so no purging is needed.

**Reply by ikhattab** [reply]
@surjithctly this is helpful, thank you 🙏🏽
